Packs/CortexXpanse/Integrations/FeedXpanse/README.md

```diff
@@ -3,6 +3,7 @@ This integration was integrated and tested with version 2.5 of Cortex Xpanse.
 
 ## Configure Xpanse Feed in Cortex
 
+
 | **Parameter** | **Description** | **Required** |
 | --- | --- | --- |
 | Server URL | The web UI with \`api-\` appended to front \(e.g., https://api-xsiam.paloaltonetworks.com\). For more information, see https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api-overview/get-started-with-cortex-xdr-apis. | True |
```

Comment:
The web UI with \`api-\` appended to the front \(for example, https://api-xsiam.paloaltonetworks.com\). For more information, see [Get started with Cortex XDR APIs](https://docs.paloaltonetworks.com/cortex/cortex-xdr/cortex-xdr-api/cortex-xdr-api-overview/get-started-with-cortex-xdr-apis).